index.html

```diff
@@ -505,32 +505,32 @@
 
-The formula for the specular IOR modulation is as follows. Given the existing **`specular_ior`**, the ratio $\eta_s$ of this to the IOR of the surrounding medium is computed (which should take into account the presence of the coat, according to equation [effective_coat_ior] in the Coat section).
+The formula for the specular IOR modulation controlled via **`specular_weight`** is as follows. Given the existing **`specular_ior`**, the ratio $\eta_s$ of this to the IOR of the surrounding medium is computed (which should take into account the presence of the coat, according to equation [effective_coat_ior] in the Coat section).
 Given this ratio, the dielectric Fresnel reflection factor of $f_\mathrm{dielectric}$ at normal incidence is given by
 \begin{equation}
 F_s = \frac{|\eta_s - 1|^2}{(\eta_s + 1)^2} \ .
 \end{equation}
-This Fresnel factor is then modulated by multiplying by twice $\xi_s = \mathtt{specular\_ior\_level}$. Thus solving for the new IOR ratio $\eta^\prime_s$ after the modulation:
+This Fresnel factor (of the initial reflection from above) is then modulated by multiplying by $\xi_s = \mathtt{specular\_weight}$. Thus solving for the new IOR ratio $\eta^\prime_s$ after the modulation:
 \begin{equation} \label{modulated_ior}
-\eta^\prime_s = \frac{1 + \mathrm{sgn}(\eta_s-1)\sqrt{2\,\xi_s F_s}} {1 - \mathrm{sgn}(\eta_s-1)\sqrt{2\, \xi_s F_s}} \ .
+\eta^\prime_s = \frac{1 + \mathrm{sgn}(\eta_s-1)\sqrt{\xi_s F_s}} {1 - \mathrm{sgn}(\eta_s-1)\sqrt{\xi_s F_s}} \ .
 \end{equation}
-Note there is a constraint $\xi_s < 1/(2 F_s)$ since the scaled reflection coefficient cannot exceed $1$. So the range of $\xi_s$ needs to be clamped: $\xi_s \in [0, \mathrm{min}(1, 1/(2 F_s)]$. Applying this modulated IOR ratio $\eta^\prime_s$ in the dielectric Fresnel factor then produces the desired reflectivity modulation at any incident angle (according to the standard formula for the dielectric Fresnel factor, assuming unpolarized light).
+For convenience, we also allow $\xi_s = \mathtt{specular\_weight}$ to exceed 1 so that the reflectivity is increased above the level set by **`specular_ior`**.
+However there is a constraint $\xi_s \le 1/F_s$ since the scaled reflection coefficient cannot exceed $1$. So the range of $\xi_s$ needs to be clamped internally: $\xi_s \in [0, \mathrm{min}(1, 1/F_s)]$. Applying this modulated IOR ratio $\eta^\prime_s$ in the dielectric Fresnel formula then produces the desired reflectivity modulation at any incident angle.
```

Comment:
👍
I had completely missed that constraint.

Reply:
Yep this constraint is important to ensure that bumping the `specular_weight` doesn't produce a non-physical Fresnel factor > 1. It means that as `specular_weight` is dialed to infinity, the Fresnel factor just saturates to 1.

Reply:
Do you like the proposal to allow `specular_weight` > 1?  (Arbitrarily high, then clamped as above).